yt_dlp/postprocessor/sponsorblock.py

```diff
@@ -68,12 +72,13 @@ def duration_filter(s):
 
         def to_chapter(s):
             (start, end), cat = s['segment'], s['category']
+            title = s['description'] if cat == 'chapter' else self.CATEGORIES[cat]
             return {
                 'start_time': start,
                 'end_time': end,
                 'category': cat,
-                'title': self.CATEGORIES[cat],
-                '_categories': [(cat, start, end)]
+                'title': title,
+                '_categories': [(title, start, end)]
```

Comment:
It doesn't really matter what this field is called, So I've reverted it to keep the diff small